include/BaslerCamera.h

```diff
@@ -176,12 +176,16 @@ class LIBBASLER_API Camera
  private:
     class _AcqThread;
     friend class _AcqThread;
+
+    Camera_t& _getBasler();
```

Comment:
This method cannot be const because it creates an AutoMutex which needs a non const mutex from the condition.
Any ideas?Agree